benchmark-service/benchmark/benchmark.py

```diff
@@ -57,6 +59,44 @@ def get_benchmarks(file_path):
     return benchmarks
 
 
+def get_comparisons(benchmarks1, benchmarks2, metric):
+    def get_metric(data):
+        if metric == 'median':
+            return data['median']
+        return min(data['times'])  # metric == 'best'
+
+    comparisons = []
+    for d in benchmarks1['data']:
+        for item in benchmarks2['data']:
+            if item['name'] == d['name']:
+                comparisons.append((d['name'], get_metric(d), get_metric(item)))
```

Comment:
Did you want an inner join here? I think it might be nice to show the outer join on metrics and have None for the comparisons if a benchmark isn't in a different benchmarks file.

Reply:
I'm thinking we can show the outer join and have a button that filters to show only the ones in common

Reply:
That would be good if it's easy to implement. Otherwise, I think the outer join is sufficient.